### PR TITLE
CNV-18014: Update Delete action for Bootable volumes

### PR DESCRIPTION
**Pull request by hstastna**
## 📝 Description

Update the _Delete_ action for volumes in _Bootable volumes_ list to delete description annotation (_metadata.annotations.description_) and both labels related to bootable volumes:
- _instancetype.kubevirt.io/default-preference_
- _instancetype.kubevirt.io/default-instancetype_

This PR is related to the feature: https://issues.redhat.com/browse/CNV-18014

The delete action:
delete_labels

## 🎥 Demo
**Before:**
Only _instancetype.kubevirt.io/default-preference_ deleted from the _DataSource_ resource, the rest still there (yaml):
yyaml_before

**After:**
Both labels and the description removed from the _DataSource_:
yyaml_after

### CODE REVIEW

**Comment by hstastna** [comment]
@metalice @pcbailey @upalatucci @vojtechszocs please review

File: src/views/bootablevolumes/actions/components/DeleteBootableVolumesModal.tsx [hunk 1]
```diff
@@ -27,12 +27,12 @@ const DeleteBootableVolumesModal: FC<DeleteBootableVolumesModalProps> = ({
...
       onClose={onClose}
-      headerText={t('Delete label?')}
-      onDeleteSubmit={deletePreferenceLabel(dataSource)}
+      headerText={t('Delete labels?')}
+      onDeleteSubmit={deleteBootableVolumeMetadata(dataSource)}
       bodyText={
         <Trans t={t}>
-          Please note that only the label data will be deleted and that the bootable volume will
-          remain. Are you sure you want to delete the label for bootable volume{' '}
+          Please note that only the labels data will be deleted and that the bootable volume will
```
Comment (pcbailey) [inline src/views/bootablevolumes/actions/components/DeleteBootableVolumesModal.tsx:34]: If I understand correctly, this action will delete both the description annotation and the default labels. Shouldn't the message indicate that the description annotation will be deleted as well?

Comment (hstastna) [reply src/views/bootablevolumes/actions/components/DeleteBootableVolumesModal.tsx:34]: This is how it was designed:
https://docs.google.com/document/d/15Cv54978AZKeukkurvkFN1YEx_kBsm_3XcmLey_cgZY/edit#heading=h.2mig9v8s38t
Note that,unfortunately, not everything is up to date in that design doc. I asked Foday to update it but it seems it takes time 🤷🏼‍♀️

Comment (pcbailey) [reply src/views/bootablevolumes/actions/components/DeleteBootableVolumesModal.tsx:34]: Understood. Let's merge it for now and we can readdress it when the design gets updated.

**Pull request merged**